README.md

```diff
@@ -7,6 +7,12 @@
 
 [Hubspot](https://www.hubspot.com/) is a marketing, sales, and service software that helps your business grow without compromise. Because “good for the business” should also mean “good for the customer.”
 
+## This library supports only legacy API
```

Comment:
This library supports only [legacy](https://legacydocs.hubspot.com/docs/overview) API. Please consider switching to the [latest](https://github.com/HubSpot/hubspot-api-php) API.

Reply:
What about ^^?